resources/views/users/admin/admin_includes/contact.blade.php

```diff
@@ -7,7 +7,13 @@
     <div class="card-body">
 
         <p class="card-text">
-            <i class="fas fa-at fa-fw me-2"></i> <a href="mailto:{{ $user->email }}">{{ $user->email }}</a><br>
+            <i class="fas fa-at fa-fw me-2"></i> <a href="mailto:{{ $user->email }}">{{ $user->email }}</a>
+            @if($user->id !== Auth::user()->id)
```

Comment:
Maybe also add a "can board" check, just to be sure .

Reply:
By reading this again I now also see a board member could update a sys admins email and get that account for themselves. I will add a check like we have with the impersonation to make it so you can only change the email from someone with the same or lower amount of rights as you.